x-pack/legacy/plugins/alerting/server/alerts_client.ts

```diff
@@ -286,10 +316,32 @@ export class AlertsClient {
       },
       { version }
     );
+
+    await this.invalidateApiKey({ apiKey: attributes.apiKey });
+  }
+
+  private async invalidateApiKey({ apiKey }: { apiKey: string | null }): Promise<void> {
+    if (!apiKey) {
+      return;
+    }
+
+    const apiKeyId = Buffer.from(apiKey, 'base64')
+      .toString()
+      .split(':')[0];
+    const response = await this.invalidateAPIKey({ id: apiKeyId });
```

Comment:
If this throws (instead of returning an `error_count > 0`) then we wo't get the error message that says we failed to invalidate the key. Is it worth catching and logging accordingly?

Reply:
The log message will still be there from the security plugin: https://github.com/elastic/kibana/blob/master/x-pack/plugins/security/server/authentication/api_keys.ts#L161.

Though I went ahead and also wrapped it ourselves to swallow the error in that case: https://github.com/elastic/kibana/pull/53732/commits/4a4aad9beb9842b2ad1aa3015ceb6b7fa4f67e71.